### PR TITLE
[testonly] CI stability experiments wrt parallel settings and TestBrokerWithManyTriggers flakiness

### PR DESCRIPTION
**Pull request by maschmid**
do not merge

### CODE REVIEW

File: vendor/knative.dev/eventing/test/rekt/features/broker/feature.go [hunk 1]
```diff
@@ -141,8 +144,10 @@ func ManyTriggers() *feature.FeatureSet {
 			}
 
 			// Install the trigger
-			f.Setup("install trigger", trigger.Install(sink, brokerName, cfg...))
-			f.Setup("trigger goes ready", trigger.IsReady(sink))
+			f.Setup("install trigger", func(ctx context.Context, t feature.T) {
```
[the remark this answers is not in the document]
Comment (maschmid) [reply vendor/knative.dev/eventing/test/rekt/features/broker/feature.go:147]: hm, right. If the broker Setup is starved, the triggers won't become ready.  I'll try to move the whole block into a a single setup feature then

[the remark this answers is not in the document]
Comment (mgencur) [reply vendor/knative.dev/eventing/test/rekt/features/broker/feature.go:147]: I'm not exactly sure which steps need to be executed serially here, maybe it makes sense to move also `eventshub.Install(sink, eventshub.StartReceiver)(ctx, t)` under `f.Setup("install trigger", func(ctx context.Context, t feature.T) {` ?

**Comment by mgencur** [comment]
Looks good now!

**Comment by mgencur** [comment]
Try OCP 4.14. I think we're hitting OCP 4.15 bug.  We have a workaround in place for s-o now: https://github.com/openshift-knative/serverless-operator/commit/fae30074535172166778fcc1bbb3bcdfb35e0309

**Pull request closed without merging**